radicle-seed/src/main.rs

```diff
@@ -0,0 +1,47 @@
+use std::net;
+
+use librad::peer::PeerId;
+use radicle_seed::{Mode, Node, NodeConfig};
+
+use argh::FromArgs;
+use futures::executor;
+use log;
+
+#[derive(FromArgs)]
+/// Radicle Seed.
+pub struct Options {
+    /// track the specified peers only
+    #[argh(option)]
+    pub track_peers: Vec<PeerId>,
```

Comment:
Should this not be `Mode`?

Reply:
Not that easy to have an enum-like interface on the cli!

Reply:
`argh` what's even the point :shrug: 

So how would start it in the `RadUrn` mode?

Reply:
Just hadn't added the option yet. It's possible now :+1: